docs/src/literate-tutorials/reactive_surface.jl

```diff
@@ -0,0 +1,259 @@
+# Putting this flag to false reproduces the figure shown in the example #src
+# We check for laminar flow development in the CI                       #src
+if @isdefined is_ci    #hide
+    IS_CI = is_ci      #hide
+else                   #hide
+    IS_CI = false      #hide
+end                    #hide
```

Comment:
Does this work? I think the flag is set in `Main` (from `make.jl`) but this is always evaluated in a separate module I think?

Reply:
Thanks for the pointer. Now this makes sense. And you are right, this is not working at all.

Reply:
I guess `isdefined(Main, :is_ci)` would work.
```suggestion
if isdefined(Main, :is_ci) #hide
    IS_CI = Main.is_ci     #hide
else                       #hide
    IS_CI = false          #hide
end                        #hide
```

Reply:
Or should we just do
```suggestion
const IS_CI = haskey(ENV, "GITHUB_ACTIONS")      #hide
```
I am fine with either.